src/formatDate.js

```diff
@@ -8,7 +8,84 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatObject = arrayInObject(fromFormat);
```

Comment:
The function `arrayInObject` seems to be unnecessary because it creates an object with the same properties as the input array. You can directly use the array elements without this conversion.